kubernetes/kserve/image_transformer/README.md

```diff
@@ -134,7 +134,7 @@ The request first comes to the image transformer at port 8080 and in turn reques
 - The curl request for inference is as below:
 
 ```
-curl -H "Content-Type: application/json" --data @serve/kubernetes/kserve/kf_request_json/v1/mnist.json http://127.0.0.1:8080/v1/models/mnist:predict
```

Comment:
I'm nor sure if this should be `0.0.0.0`

Reply:
For KServe the steps for getting Ingress host and port and documented here, please use the same steps: 

https://kserve.github.io/website/0.11/modelserving/v1beta1/torchserve/#model-inference_1